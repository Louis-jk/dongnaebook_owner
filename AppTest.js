import React from 'react'
import { View, Text } from 'react-native'
import { TextInput } from 'react-native-gesture-handler'

const AppTest = () => {
  const onChangeText = (evt) => {
    console.log('onChangeText evt', evt)
  }

  return (
    <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
      <Text>테스트 컴포넌트</Text>
      <TextInput placeholder='Enter data' onChangeText={onChangeText} />
    </View>
  )
}

export default AppTest
