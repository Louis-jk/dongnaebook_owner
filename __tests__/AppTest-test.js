import * as React from 'react'

import AppTest from '../AppTest'
import { render } from '@testing-library/react-native'
import renderer from 'react-test-renderer'

test('AppTest jest test', () => {
  const tree = renderer.create(<AppTest />).toJSON()
  expect(tree).toMatchSnapshot()
})
