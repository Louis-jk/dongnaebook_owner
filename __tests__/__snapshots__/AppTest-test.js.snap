// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppTest jest test 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    테스트 컴포넌트
  </Text>
  <TextInput
    allowFontScaling={true}
    collapsable={false}
    onChangeText={[Function]}
    onGestureHandlerEvent={[Function]}
    onGestureHandlerStateChange={[Function]}
    placeholder="Enter data"
    rejectResponderTermination={true}
    underlineColorAndroid="transparent"
  />
</View>
`;
